proxy/api/src/config.rs

```diff
@@ -27,6 +27,18 @@ pub fn dirs() -> ProjectDirs {
     ProjectDirs::from("xyz", "radicle", "radicle-upstream").expect("couldn't build dirs")
 }
 
+/// Returns the directory for the application store
+pub fn store_dir() -> path::PathBuf {
+    let store_root = match std::env::var_os("RAD_HOME") {
+        None => {
+            let dirs = dirs();
+            return dirs.data_dir().to_path_buf();
```

Comment:
Shouldn’t we also join this with a `store` segment?

Reply:
`store` is joined at the end in both cases in [l.39](https://github.com/radicle-dev/radicle-upstream/pull/1527/files/0c8d50f0b5aef28081168663b39bc8501e4bdd6d#diff-c829644a81bfa71911f54a78e6369e3d4279dcb077586f7b47d92374213d6aeeR39)

Reply:
Are you sure? We `return` here.

Reply:
Oh nice catch :smile:

Reply:
🤦 👍